cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -990,10 +990,10 @@ def test_datepicker_in_service_request():
     pass
 
 
-@pytest.mark.meta(coverage=[1740823])
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_dialog_dropdown_integer_required():
+@pytest.mark.meta(automates=[1740823])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("file_name", ["bz_1740823.yml"], ids=["sample_dialog"],)
+def test_multi_dropdown_dialog_value(appliance, generic_catalog_item_with_imported_dialog):
```

Comment:
You forgot to add `file_name` parameter here.

Reply:
@valaparthvi here file_name only using for uploading the dialog. in this test we don't need it.